packages/main/src/plugin/image-registry.ts

```diff
@@ -921,20 +924,24 @@ export class ImageRegistry {
   }
 
   async searchImages(options: ImageSearchOptions): Promise<ImageSearchResult[]> {
-    if (!options.registry) {
-      options.registry = 'https://index.docker.io';
-    }
-    if (options.registry === 'docker.io') {
-      options.registry = 'index.docker.io';
-    }
-    if (!options.registry.startsWith('http')) {
-      options.registry = 'https://' + options.registry;
+    try {
+      if (!options.registry) {
+        options.registry = 'https://index.docker.io';
```

Comment:
(Future PR) Should we have this in a configuration value, like `default registry` ?